dpp4/core/basyx_adapter.py

```
@@ -1,616 +1,1051 @@
 """
 
-from typing import Dict, List, Any, Optional, Tuple, Union
+from typing import (
+    Dict,
+    Any,
+    List,
+    Optional,
+    Union,
+    BinaryIO,
+    TextIO,
+    TYPE_CHECKING,
```

Ruff (0.8.2)

`typing.TYPE_CHECKING` imported but unused

Remove unused import

(F401)